pkg/ingester/ingester.go

```diff
@@ -701,7 +712,7 @@ func (i *Ingester) PushWithCleanup(ctx context.Context, pushReq *push.Request) (
 
 		// Keep track of some stats which are tracked only if the samples will be
 		// successfully committed
-		stats pushStats
+		stats, ephemeralStats pushStats
```

Comment:
[nit] What if we rename `stats` to `persistentStats`? Is that how we're going to call the standard TSDB storage, right?